packages/theme/src/Theme/colors.ts

```diff
@@ -43,6 +43,7 @@ export const MaskColors = {
             publicSecond: '#767F8D',
             publicThird: '#ACB4C1',
             publicLine: '#F2F5F6',
+            publicTwitter: '#1D9BF0',
```

Comment:
@nuanyang233

Reply:
This color does not exist in the ui kit, please check it.

Reply:
![image](https://user-images.githubusercontent.com/72531217/191007503-020256cc-8ef5-4bea-a8c9-a3d20c1b76e5.png)